instrument/SXD_Definition.xml

```diff
@@ -189,7 +200,7 @@ bank1,bank2,bank3,bank4,bank5,bank6,bank7,bank8,bank9,bank10,bank11
   <type is="detector" name="pixel">
     <cuboid id="pixel-shape">
       <left-front-bottom-point y="-0.0015" x="-0.0015" z="0.0015"/>
-      <left-front-top-point y="0.00015" x="-0.0015" z="0.0015"/>
+      <left-front-top-point y="0.0015" x="-0.0015" z="0.0015"/>
```

Comment:
Good spot - as discussed this affects any solid angle calculation (although luckily we rely on the vanadium for this atm!)